source/api/commands/as.md

```diff
@@ -70,6 +91,47 @@ cy.wait('@userPut')
 
 Some strings are not allowed as alias names since they are reserved words in Cypress. These words include: `test`, `runnable`, `timeout`, `slow`, `skip`, and `inspect`.
 
+## `as` is asynchronous
+
+Remember that **Cypress commands are async**, including `as()`.
+
+Because of this you cannot _synchronously_ access anything you have aliased. You must use other asynchronous commands such as {% url `.then()` then %} to access what you've aliased.
+
+Here are some further examples of using `as()` that illustrate the asynchronous behaviour.
+
+```javascript
+describe('A fixture', () => {
+
+  describe('alias can be accessed', () => {
+    it('via get().', () => {
+      cy.fixture('admin-users.json').as('admins');
```

Comment:
We also do not use semicolons in our code examples - again, need to write up our style guide 😅

Reply:
> I need to write up our style guide.

I'm not being cheeky here... shall I write a guide and then you can ~tear it to pieces~ review it? 😄Cypress is doing great work for us and I'd like to contribute back. If me writing that would be unhelpful then no worries.